pkg/ebpf/c/common/consts.h

```diff
@@ -61,4 +64,7 @@ enum buf_idx_e
     MAX_BUFFERS
 };
 
+#define BPF_PROG_LOAD 5
```

Comment:
I originally aggregated all consts here as a convenience, but I believe we should overtime migrate the consts to more domain appropriate files (I think some consts already are).
These fit better in the new `bpf.h` file IMO, WDYT?